tests/hardhat/Comptroller/accountLiquidityTest.ts

```diff
@@ -28,9 +28,11 @@ interface AccountLiquidityTestFixture {
 async function makeComptroller(): Promise<AccountLiquidityTestFixture> {
   const poolRegistry = await smock.fake<PoolRegistry>("PoolRegistry");
   const accessControl = await smock.fake<AccessControlManager>("AccessControlManager");
-  const comptrollerFactory = await smock.mock<Comptroller__factory>("Comptroller");
-  const comptroller = await comptrollerFactory.deploy(poolRegistry.address, accessControl.address);
-  await comptroller.initialize();
+  const Comptroller = await smock.mock<Comptroller__factory>("Comptroller");
```

Comment:
I haven't pushed my changes but essentially I broke this function and others out into its own file so that we don't have it duplicated in ever test file